components/ILIAS/Saml/classes/class.ilSamlSettingsGUI.php

```diff
@@ -126,7 +128,7 @@ public function __construct(private readonly int $ref_id)
         $this->ctrl = $DIC->ctrl();
         $this->tpl = $DIC->ui()->mainTemplate();
         $this->lng = $DIC->language();
-        $this->access = $DIC->access();
+        $this->rbac_system = $DIC->rbac()->system();
```

Comment:
You already have the complete RBAC services assigned to a class member one line below.